src/openapi.json

```diff
@@ -0,0 +1,177 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "Philanthropy Data Commons API",
+    "description": "An API for a common data platform to make the process of submitting data requests to funders less burdensome for organizations seeking grants.",
+    "version": "0.0.5"
```

Comment:
Since the other iterations weren't merged idk if they count as taking up a version number -- fine merging this as is but if you wanna make this 0.0.1 that'd be reasonable IMHO (and in future / once this is "released" I don't think we should jump multiple versions just from branch iteration)

Reply:
So versions 0.0.1 and 0.0.2 were force-pushed out of the git history, but versions 0.0.3 and 0.0.4 are still there in the branch (and therefore will be in the eventual history). There really were five versions published (if you count this PR as publication) which was why I erred on the safe side.

Is this something that should be squashed, and then if so maybe we could call it 0.1.0?